### DISCUSSION TITLE
Clustering and Custom App Ingestion

### DISCUSSION THREAD

**Discussion by CrossBound** [datalust/seq-tickets#1442] [General]
I'm considering running a custom app in Seq that will ingest logs from a message queue. If I have a DR cluster set up with the same ingestion app running on both of the cluster nodes, will it synchronize those logs between the nodes or does the synchronizing only occur if the logs are ingested through the Seq HTTP API?

**Comment by nblumhardt** [comment]
Hi David-

Logs ingested into a Seq DR configuration will be replicated to both nodes regardless of whether that is through an app or the HTTP API, so this should be fine.

Hope this helps,
Nick
